python/shotgun_fields/shotgun_field_editable.py

```diff
@@ -98,6 +98,13 @@ def _on_current_changed(self, index):
 
         self.currentWidget().setFocus()
 
+    def __getattr__(self, name):
+        """
+        Routes any attributes not found on the editable widget to the
+        fields widget that it is wrapping.
+        """
+        return getattr(self._display.display_widget, name)
```

Comment:
Maybe the methods do need to be handled individually. I think we need to forward `value_changed` signals from each of `editor` and `display`. 

`get_value` and `set_value` calls should probably update the values of the currently displayed widget. 

Would be nice to implement `enable_editing` to allow callers to programmatically swap the state. That method is required for widgets that are both display and edit so it would provide nice consistency.